docs/query_engine/config/annotated_config.md

```diff
@@ -0,0 +1,3 @@
+# Annotated Config
```

Comment:
Would rather have this under an existing heading rather than a new page for one line; alternatively add the annotated config here

Reply:
Where do you suggest?

Reply:
I think we should keep this here. I agree it's not the prettiest thing, but I think it makes sense.

Reply:
I'd rather just have the config straight up written here

I think you can do something  like this to format it well:
`/``yaml
config:
 blah blah blah
`/``

Reply:
Yeah it just doesn't look as good. @andrewmains12 tiebreaker?

Reply:
You don't get any of the coloring I don't think

Reply:
<img width="793" alt="Screen Shot 2019-03-18 at 3 29 32 PM" src="https://user-images.githubusercontent.com/4688976/54557922-b1e2e280-4992-11e9-9fb4-6ef21a87cc36.png">

Just looks like this on our docs..

Reply:
Personally I prefer the approach @benraskin92 has here; having it in a `.yaml` makes it easy read + edit (yes, I know you can change the file association etc, but it's a pain). If we decide it's bothersome later we can always change it back.